provision/initramfs/Makefile.am

```diff
@@ -145,6 +145,8 @@ rootfs: busybox e2fsprogs xfsprogs libarchive parted curl
 	mkdir rootfs/usr
 	mkdir rootfs/lib
 	mkdir rootfs/lib64
+	ln -s ../bin rootfs/usr/bin
```

Comment:
Maybe I'm missing something here... but isn't this creating a symlink *at*  rootfs/usr/bin pointing to ../bin ? 

And if /bin, /sbin are the links ... why would you make rootfs/usr/bin the link?